.github/Pull_request_Template.md

```diff
@@ -0,0 +1,17 @@
+# Related Issuse or bug
+  - Info about Issue or bug
+
+# Fixes
+  - [issue number that will be closed through this PR]
```

Comment:
Hey @ritiksoni00 please provide this full line here `Fixes: #[issue number that will be closed through this PR]`